### PR TITLE
Add ContentView with Crawler class

### PR DESCRIPTION
**Pull request by Juung**
3 methods in Crawler: getTitle(), getContent(), and getWordCount()
minor issues
1. getContent(): no images
2. ContentView: no title, can't scroll pageView

### CODE REVIEW

**Comment by Csoyee** [comment]
2. 
You mean you don't have scroll that you can't see the long text by text view? 
> http://digapps.tistory.com/entry/%EC%95%88%EB%93%9C%EB%A1%9C%EC%9D%B4%EB%93%9C-TextView-%EC%8A%A4%ED%81%AC%EB%A1%A4-%EB%84%A3%EA%B8%B0
Above link will help!

And maybe we can separate text view into two (for title and for content).

File: Roys/app/src/main/java/com/dwg_karrier/roys/ContentView.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+public class ContentView extends AppCompatActivity {
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.content);
+    try {
+      /* TODO
+      * bring user-clicked URL
+      * merge with soyee's code
+      */
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/ContentView.java]: nit: https://google.github.io/styleguide/javaguide.html#s4.8.6-comments

File: Roys/app/src/main/java/com/dwg_karrier/roys/ContentView.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      // yet, write exact pageUrl
+      String pageUrl = "https://mercury.postlight.com/parser?url=http://www.bloter.net/archives/265787";
+      Crawler crawler = new Crawler(pageUrl);
+      // show title and content in one page
+      // It's better to scroll down in concatenated version
+      String title = crawler.getTitle();
+      String content = crawler.getContent();
+      TextView contentView = (TextView) findViewById(R.id.contentView);
+      contentView.setText(title+"\n\n"+content);
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/ContentView.java]: nit: https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
Did the Google Java convention plugin applied? I believe this type of convention should be applied automatically.

File: Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/* Add on to listActivity
+* Use getTitle(), getContent(), getWordCount() */
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java]: https://google.github.io/styleguide/javaguide.html#s7-javadoc
Unlike the usual comments, there's a comment with certain format called Javadoc.
It's usually used on description of a class or method, and has the bit different shape as follows.
Upper link is about the Javadoc style in Google, and here's the Wikipedia link for the real Javadoc: https://en.wikipedia.org/wiki/Javadoc

File: Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+      e.printStackTrace();
+    }
+    return jsonObject;
+  }
+
+  public String getTitle() {
+    if (this.jsonObject == null) {
+      return null;
+    }
+    String title;
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java]: The scope of title is so wide while it's modification job is only occurred within the try block.
My recommendation to narrow down the variable scope is,
`try {`
`return (String) this.jsonObject.get("title");`
`} catch (Exception e) {`
`  ...`
`  return null;`
`}`

File: Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+      MyAsyncTask myAsyncTask = new MyAsyncTask();
+      jsonObject = myAsyncTask.execute(pageUrl).get();
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+    return jsonObject;
+  }
+
+  public String getTitle() {
+    if (this.jsonObject == null) {
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java]: Is `this.` required?

File: Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java (hunk 4 above)
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java]: or if you want to concentrate the return logic into a single line,
`String title;`
`try {`
`  title = something`
`} catch (Exception e) {`
`  ...`
`  title = null;`
`}`
`return title;`
might be possible.

File: Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+      Log.e("getWordCount_Error:", e.getMessage(), e);
+      return null;
+    }
+    return wordCount;
+  }
+
+  public class MyAsyncTask extends AsyncTask<String, Void, JSONObject> {
+    @Override
+    protected JSONObject doInBackground(String... strings) {
+      JSONObject json = null;
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java]: Variable `json` was declared too early without any use at here.

File: Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+          br.close();
+        }
+        conn.disconnect();
+      } catch (Exception e) {
+        Log.e("Error", e.getMessage(), e);
+        return null;
+      }
+      try {
+        String jsonStr = stringBuilder.toString();
+        JSONParser jsonParser = new JSONParser();
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java]: `jsonStr` and `jsonParser` doesn't need to be here; the code still make sense if two variables' declaration is moving into the below `try` statement.

**Pull request merged**